src/sage/graphs/generators/basic.py

```diff
@@ -1149,6 +1149,89 @@ def LadderGraph(n):
     return G
 
 
+def MoebiusLadderGraph(n):
+    r"""
+    Return a Möbius ladder graph with `2n` nodes
+
+    A Möbius ladder graph of order `2n` is a ladder graph of the same order
+    that is connected at the ends with a single twist, i.e., a ladder graph
+    bent around so that top meets bottom with a single twist. Alternatively,
+    it can be described as a single cycle graph (of order `2n`) with the
+    addition of edges (called `rungs`) joining the antipodal pairs of nodes.
+    Also, note that the Möbius ladder graph ``graphs.MoebiusLadderGraph(n)`` is
+    precisely the same graph as the circulant graph
+    ``graphs.CirculantGraph(2 * n, [1, n])``.
+
+    PLOTTING:
+
+    Upon construction, the position dictionary is filled to override the
+    spring-layout algorithm. By convention, each Möbius ladder graph will be
+    displayed with the first (0) node at the top, with the rest following in a
+    counterclockwise manner.
+
+    INPUT:
+
+    - ``n`` -- a non negative integer; number of nodes is `2n`
+
+    OUTPUT:
+
+    - ``G`` -- a Möbius graph of order `2n`; note that a :class:`ValueError` is
+      returned if ``n < 0``
+
+    EXAMPLES:
+
+    Construct and show a Möbius ladder graph with 26 nodes::
+
+        sage: g = graphs.MoebiusLadderGraph(13)
+        sage: g.show()  # long time  # needs sage.plot
+
+    Create several Möbius ladder graphs in a Sage graphics array::
+
+        sage: g = []
+        sage: j = []
+        sage: for i in range(9):
+        ....:    k = graphs.MoebiusLadderGraph(i+3)
+        ....:    g.append(k)
+        sage: for i in range(3):  # needs sage.plot
```

Comment:
here you can use block scope for `# needs sage.plots`, that is:
```
sage: # needs sage.plots
sage: g = []
...
```

Reply:
Is it fine if for this one also we go with `# needs sage.plots` mentioned at column no. 88 like the other ones (similar to the way it is mentioned here: [`graphs.CircularLadderGraph()`](https://doc.sagemath.org/html/en/reference/graphs/sage/graphs/graph_generators.html#sage.graphs.graph_generators.GraphGenerators.CircularLadderGraph)?
In other words, at line no. 1195 we will have the following:
```
sage: for i in range(3):                                                        # needs sage.plot
```

---

PS: In the first commit, I mentioned each of  `# needs sage.plots` at column no. 88 analogously the way it is written here:  [`graphs.CircularLadderGraph()`](https://doc.sagemath.org/html/en/reference/graphs/sage/graphs/graph_generators.html#sage.graphs.graph_generators.GraphGenerators.CircularLadderGraph), but in the [general convention](https://doc.sagemath.org/html/en/developer/coding_basics.html#python-code-style), I found the following line pertaining to sage documentation style:


*Lines should be shorter than 80 characters. If in doubt, read [PEP8: Maximum Line Length](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).*

That's the reason, I changed those. Anyways, thanks for the info. 😊👍

Reply:
The `# needs ...` tag is special and we try to minimize the impact on readability. In general we place it at column 88.
When we have a block of doctests with multiple time `# needs sage.plots`, we can use block scope, that is start the block of doctests with `sage: # needs sage.plots` and we don't add extra spaces. See for instance https://doc.sagemath.org/html/en/reference/graphs/sage/graphs/graph_generators.html#sage.graphs.graph_generators.GraphGenerators.CompleteBipartiteGraph